deps/wazuh_testing/wazuh_testing/qa_ctl/run_tests/Pytest.py

```diff
@@ -114,34 +122,30 @@ def run(self, ansible_inventory_path, custom_report_file_path=None):
         fetch_html_report = {'fetch': {'src': os.path.join(self.tests_run_dir, html_report_file_name),
                                        'dest': self.tests_result_path, 'flat': 'yes'}}
 
-        ansible_tasks = [AnsibleTask(execute_test_task), AnsibleTask(create_plain_report),
-                         AnsibleTask(fetch_plain_report), AnsibleTask(fetch_html_report)]
-
-        playbook_parameters = {'become': True, 'tasks_list': ansible_tasks, 'playbook_file_path':
-                               '/tmp/playbook_file.yaml', "hosts": self.hosts}
+        create_assets_directory = {'local_action': {'module': 'ansible.builtin.file',
+                                                    'path': os.path.join(self.tests_result_path, assets_folder),
+                                                    'state': 'directory'},
+                                   'become': False}
```

Comment:
If I remove that line then the task will fail because it needs a password for the localhost. I have set become:true at a higher level in playbook parameters, so I need to specify become:false at those tasks that run in localhost or look for other solutions.